pages/start_page.py

```diff
@@ -5,17 +5,21 @@
 # file, You can obtain one at http://mozilla.org/MPL/2.0/.
 
 from selenium.webdriver.common.by import By
+from selenium.webdriver.support import expected_conditions as expected
 from selenium.webdriver.support.ui import WebDriverWait
 
 from pages.base import Base
 
-
```

Comment:
We should restore this as it's a PEP8 rule. Top level definitions should be separated by two blank lines.